CHANGELOG.md

```diff
@@ -8,6 +8,9 @@
 * Update default node-version for GitHub Action generator and fixes PG setup issues. ([@abhaynikam][])
 * Updates Bootstrap generator for supporting Bootstrap 5 with popper.js. ([@abhaynikam][])
 
+## 0.12.0 (March 29th, 2023)
+* Adds Overcommit generator for RuboCop. ([@coolprobn][])
```

Comment:
can you please move this changelog entry to the unreleased section on line number 9? Usually, when I release the new gem version, I make sure to update the changelog with the release date.

Reply:
Makes sense, I will update it.